steamclock/steamclog-android
# Remove Firebase (Crashlytics and Analytics) (#62)

### Related Issue: #62 


### Summary of Problem:
* Issues seen trying to support both Crashlytics and Sentry at the same time
* We are moving to Sentry on all our projects now
* iOS library removed support for Firebase Analytics when they ported over to using Sentry

### Proposed Solution:
* Removed all Firebase usage

### Testing Steps:
* Run sample app
* Make sure analytic test says "not supported"
* Change log level to Release+, log things and then fire off non-fatal
* Make sure they are logged to the Sentry dashboard for the test app: https://sentry.io/organizations/steamclock-software/issues/?project=5399932
